frontends/web/src/routes/buy/info.tsx

```diff
@@ -87,7 +87,7 @@ class BuyInfo extends Component<Props, State> {
         )
         .then(results => results.filter(result => result))
         // @ts-ignore
-        .then(accounts => accounts.map(({ name, code }) => ({ text: name, value: code })))
+        .then(accounts => accounts.map(({ isToken, name, coinName, code }) => ({ text: isToken || name === coinName ? name : `${name} (${coinName})`, value: code })))
```

Comment:
Some parentheses would help readability and the intention of the code see the difference:
- `` ({ text: isToken || name === coinName ? name : `${name} (${coinName})`, value: code }) ``
- `` ({ text: (isToken || name === coinName) ? name : `${name} (${coinName})`, value: code }) ``
- `` ({ text: isToken || (name === coinName ? name : `${name} (${coinName})`), value: code }) ``

Reply:
i am an opponent of parens that are not required :O imho everyone should be familiar with precedence rules.

Reply:
I prefer readable code, when you write math you also write parentheses even tho it is not needed right?